src/ngx_http_lua_ssl_certby.c

```diff
@@ -1367,7 +1367,7 @@ ngx_http_lua_ffi_ssl_verify_client(ngx_http_request_t *r, void *ca_certs,
     /* set CA chain */
 
     if (chain != NULL) {
-        ca_store = SSL_CTX_get_cert_store(SSL_get_SSL_CTX(ssl_conn));
+        ca_store = X509_STORE_new();
         if (ca_store == NULL) {
             *err = "SSL_CTX_get_cert_store() failed";
             return NGX_ERROR;
```

Comment:
Remember to `goto failed` when failed to create `name_chain`:
https://github.com/openresty/lua-nginx-module/blob/6b58760fe4da41c895ab8dc4ddbd8c5feccc75f2/src/ngx_http_lua_ssl_certby.c#L1378-L1381

Reply:
Fixed.